python/tvm/micro/testing/evaluation.py

```diff
@@ -32,20 +32,30 @@
 
 
 def tune_model(
-    platform, board, target, mod, params, num_trials, tuner_cls=tvm.autotvm.tuner.GATuner
+    platform,
+    board,
+    target,
+    mod,
+    params,
+    num_trials,
+    tuner_cls=tvm.autotvm.tuner.GATuner,
+    options=None,
```

Comment:
nit: this argument is called `project_options` (instead of just `options`) in the `evaluate_model` function below. Can we standardize this argument across both functions?